File: app/src/main/java/com/igtools/downloader/models/Record.java
package com.igtools.downloader.models;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.util.Date;

@Entity
public class Record {
    @PrimaryKey(autoGenerate = true)
    public int id;
    @ColumnInfo(name = "content")
    public String content;
    @ColumnInfo(name = "create_time")
    public String createTime;
}

File: app/src/main/java/com/igtools/igdownloader/models/Record.kt
package com.igtools.igdownloader.models

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.util.*

@Entity(tableName = "record")
data class Record(
    @PrimaryKey
    var id: String,
    @ColumnInfo(name = "content")
    var content: String?,
    @ColumnInfo(name = "created_time")
    var createdTime: Long?

)




